tests/framework/TestResources.php

```diff
@@ -52,6 +52,7 @@ class TestResources
     const VALUE2        = 'value2';
     const VALUE3        = 'value3';
     const ACCOUNT_NAME  = 'myaccount';
+    const SAS_TOKEN     = 'st=2016-12-01T19%3A43%3A00Z&se=2016-12-02T19%3A43%3A00Z&sp=rwdl&sv=2015-12-11&sr=c&sig=mtG3Ykj7tC9zNS3fyYzgV8X8k5G4aCyqiAdoLPmBlvs%3D';
```

Comment:
It is a real token but using an old key (it's been changed since then) so shouldn't be a problem ?